src/ucs/time/time.h

```diff
@@ -139,6 +139,12 @@ static inline void ucs_sec_to_timeval(double seconds, struct timeval *tv)
     tv->tv_usec = usec % UCS_USEC_PER_SEC;
 }
 
+/* Convert POSIX timeval to seconds */
+static inline double ucs_timeval_to_sec(const struct timeval *tv)
```

Comment:
need to remove `ucs_timeval_to_sec` and `ucs_timespec_to_sec` now

Reply:
yes, agreed - thank you
will remove